.github/workflows/ci.yml

```diff
@@ -5,9 +5,9 @@ name: Java CI with Gradle
 
 on:
   push:
-    branches: [ master ]
+    branches: [ $default-branch ]
   pull_request:
-    branches: [ master ]
+    branches: [ $default-branch ]
```

Comment:
I had many issues with this notation and I just found out that workflow is not executed again